src/wp-includes/ms-files.php

```diff
@@ -17,8 +18,6 @@
 
 ms_file_constants();
 
-error_reporting( 0 );
```

Comment:
There's a history of removing this line and it getting added back:
https://core.trac.wordpress.org/ticket/11742#comment:22

Reply:
Do you think https://core.trac.wordpress.org/ticket/53493#comment:1 is a correct interpretation? It seems like this PR would preserve the intended behavior while fixing the unintended behavior.

Reply:
Yes. My interpretation is the same as yours. 👍